modular_skyrat/modules/job_access_overrides/code/station_engineer.dm

```diff
@@ -0,0 +1,17 @@
+/datum/id_trim/job/station_engineer
```

Comment:
this doesn't need to be its own module, `master_files` exists for a reason

Reply:
i dunno, i think it's better from a modularization standpoint to keep big sweeping changes to core code in master_files if we can modularize it, and put individual changes in the skyrat modules for easier searching given the volume of changes you guys have

Reply:
Not going to put this in `master_files` unless it's really wanted, it will make the change far harder to find. Plus, in the past, we had a ton of job access overrides, so I feel it'd just be better to get a module set up for if that were to ever happen.